targets/ramips-mt7621

```diff
@@ -126,6 +126,12 @@ device('xiaomi-mi-router-4a-gigabit-edition', 'xiaomi_mi-router-4a-gigabit', {
 	factory = false,
 })
 
+
```

Comment:
```suggestion
```

Minor nitpick: We've got two empty lines between diffrent Vendors but only one line between devices of the same vendor.